dags/event_ping_events.py

```diff
@@ -12,6 +12,7 @@
     'owner': 'ssuh@mozilla.com',
     'depends_on_past': False,
     'start_date': datetime(2018, 11, 26),
+    'end_date': datetime(2019, 12, 02),
```

Comment:
`SyntaxError: leading zeros in decimal integer literals are not permitted`
```suggestion
    'end_date': datetime(2019, 12, 2),
```

Reply:
...What is the airflow CI even doing if it's not catching syntax errors?

Reply:
😭